nsv13/code/modules/munitions/munitions_trolley.dm

```diff
@@ -0,0 +1,105 @@
+/obj/structure/munitions_trolley
+	name = "Munitions trolley"
+	icon = 'nsv13/icons/obj/munitions.dmi'
+	icon_state = "trolley"
+	desc = "A large trolley designed for ferrying munitions around. It has slots for traditional ammo magazines as well as a rack for loading torpedoes. To load it, click and drag the desired munition onto the rack."
+	anchored = FALSE
+	density = TRUE
+	layer = 3
+	var/capacity = 0 //Current number of munitions we have loaded
+	var/max_capacity = 4//Maximum number of munitions we can load at once
+	var/loading = FALSE //stop you loading the same torp over and over
+
+/obj/structure/munitions_trolley/Moved()
+	. = ..()
+	if(has_gravity())
+		playsound(src, 'sound/effects/roll.ogg', 100, 1)
+
+/obj/structure/munitions_trolley/AltClick(mob/user)
+	. = ..()
+	add_fingerprint(user)
+	if(!anchored)
+		to_chat(user, "<span class='notice'>You toggle the brakes on [src], fixing it in place.</span>")
+		anchored = TRUE
+	else
+		to_chat(user, "<span class='notice'>You toggle the brakes on [src], allowing it to move freely.</span>")
+		anchored = FALSE
+
+/obj/structure/munitions_trolley/examine(mob/user)
+	. = ..()
+	if(anchored)
+		. += "<span class='notice'>[src]'s brakes are enabled!</span>"
+
+/obj/structure/munitions_trolley/MouseDrop_T(obj/structure/A, mob/user)
+	. = ..()
+	if(istype(A, /obj/item/ship_weapon/ammunition/torpedo))
+		if(loading)
+			to_chat(user, "<span class='notice'>You're already loading something onto [src]!.</span>")
+			return
+		if(capacity < max_capacity)
+			to_chat(user, "<span class='notice'>You start to load [A] onto [src]...</span>")
+			loading = TRUE
+			if(do_after(user,20, target = src))
+				load_trolley(A, src)
+				to_chat(user, "<span class='notice'>You load [A] onto [src].</span>")
+				loading = FALSE
+			loading = FALSE
+		else
+			to_chat(user, "<span class='warning'>[src] is fully loaded!</span>")
+
+/obj/structure/munitions_trolley/proc/load_trolley(atom/movable/A, mob/user)
+//	if(istype(A, /obj/item))
```

Comment:
If this section of commented out code isn't planned on being used could you remove it? I dont know if you added that or someone else or if its important for some reason.

Reply:
Was already commented out when I got to the code. Removing.